lxd/networks.go

```diff
@@ -1023,6 +1023,66 @@ func doNetworkGet(s *state.State, r *http.Request, allNodes bool, requestProject
 	return apiNet, nil
 }
 
+// handleNetworkDelete provides the logic for deleting a network.
+func handleNetworkDelete(s *state.State, n network.Network, effectiveProjectName string, networkName string, details networkDetails, clientType clusterRequest.ClientType, clusterNotification bool, requestor *api.EventLifecycleRequestor) error {
```

Comment:
you have access to the network name via n.Name() so why do you need the `networkName` arg too?